sdk/cognitiveservices/textanalytics/Azure.AI.TextAnalytics/samples/Sample4_AnalyzeSentimentBatch.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.Testing;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace Azure.AI.TextAnalytics.Samples
+{
+    [LiveOnly]
+    public partial class TextAnalyticsSamples
+    {
+        [Test]
+        public void AnalyzeSentimentBatch()
```

Comment:
We have two files with prefix `Sample4_`. Is that correct?

Reply:
KeyVault does this as well.